tensorflow-estimator/meta.yaml

```diff
@@ -1,16 +1,17 @@
-{% set version = "2.6.0" %}
+{% set name = "tensorflow-estimator" %}
+{% set version = "2.8.0" %}
 
 package:
-  name: tensorflow-estimator
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/py2.py3/t/tensorflow-estimator/tensorflow_estimator-{{ version }}-py2.py3-none-any.whl
-  sha256: cf78528998efdb637ac0abaf525c929bf192767544eb24ae20d9266effcf5afd
+  sha256: bee8e0520c60ae7eaf6ca8cb46c5a9f4b45725531380db8fbe38fcb48478b6bb
 
 build:
   number: 0
-  noarch: python
+  skip: True  # [py<37]
   script:
     - python -m pip install --no-deps --ignore-installed tensorflow_estimator-{{ version }}-py2.py3-none-any.whl
 
```

Comment:
this package depends on tensorflow runtime dependencies, see https://github.com/tensorflow/estimator/blob/8863423c9421bfdee058252db7387397d99d0f8f/tensorflow_estimator/tools/pip_package/setup.py#L35

Reply:
Right! Let's fix this by depending on tensorflow-base.